File: car-racing-tournament-api/Models/Statistics.cs
using System.Text.Json.Serialization;

namespace car_racing_tournament_api.Models
{
    public class Statistics {
        public int? NumberOfRace { get; set; }
        public int? NumberOfWin { get; set; }
        public int? NumberOfPodium { get; set; }
        public int? NumberOfChampion { get; set; }
        public int? SumPoint { get; set; }
        public List<SeasonStatistics>? SeasonStatistics { get; set; }
        public List<PositionStatistics>? PositionStatistics { get; set; }
    }

    public class SeasonStatistics {
        public Guid Id { get; set; }
        public string? Name { get; set; }
        public int? Position { get; set; }
    }

    public class PositionStatistics {
        public string? Position { get; set; }
        public int? Number { get; set; }
    }
}

File: car-racing-tournament-web/src/app/models/statistics.ts
export class Statistics {
  constructor(
    public numberOfRace?: number,
    public numberOfWin?: number,
    public numberOfPodium?: number,
    public numberOfChampion?: number,
    public sumPoint?: number,
    public seasonStatistics?: SeasonStatistics[],
  ) {}
}

export class SeasonStatistics {
  constructor(
    public id?: string,
    public name?: string,
    public position?: number
  ) {}
}

export class PositionStatistics {
  constructor(
    public position?: string,
    public number?: number
  ) {}
}
